doc/api/async_hooks.md

```diff
@@ -203,13 +203,16 @@ UDPSENDWRAP, UDPWRAP, WRITEWRAP, ZLIB, SSLCONNECTION, PBKDF2REQUEST,
 RANDOMBYTESREQUEST, TLSWRAP
 ```
 
+There is also the `PROMISE` resource type, which is used to track `Promise`
```

Comment:
I think we should delete the list. The user shouldn't really on a defined list of types because the Embedder API allows for extra types.